client-app/core/enums/validation-error-object-type.enum.ts

```diff
@@ -0,0 +1,7 @@
+export enum ValidationErrorObjectType {
+  CartProduct = "CartProduct",
+  CatalogProduct = "CatalogProduct",
```

Comment:
https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/7259fbd827596060f22d44e4a3fa22a40f34f93c/client-app/core/constants/products.ts#L11 replace and remove please, it's used for the same reason. Thank you for the enum

Reply:
I would name it ValidationErrorType

Reply:
> I would name it ValidationErrorType

I keep original class name for naming in this case.